apps/AEPSampleAppNewArchEnabled/app/_layout.tsx

```diff
@@ -0,0 +1,102 @@
+import { DarkTheme, DefaultTheme, ThemeProvider } from '@react-navigation/native';
+import { useFonts } from 'expo-font';
+import { Stack } from 'expo-router';
+import * as SplashScreen from 'expo-splash-screen';
+import { useEffect } from 'react';
+import 'react-native-reanimated';
+import CoreView from './CoreView';
+import {Button, View} from 'react-native';
+import {NavigationProps} from '../types/props';
+import {createDrawerNavigator} from '@react-navigation/drawer';
+import {NavigationContainer} from '@react-navigation/native';
+import { Drawer } from 'expo-router/drawer';
+
+import { useColorScheme } from '@/hooks/useColorScheme';
+
+// Prevent the splash screen from auto-hiding before asset loading is complete.
+SplashScreen.preventAutoHideAsync();
+
+
+// function HomeScreen({navigation}: NavigationProps) {
```

Comment:
Can we remove these lines of code if they are not being used?

Reply:
yes have removed it now